examples/flutter_gallery/lib/demo/cupertino/cupertino_refresh_demo.dart

```diff
@@ -90,50 +93,226 @@ class _CupertinoRefreshControlDemoState extends State<CupertinoRefreshControlDem
+  ],
+  <String>[
+    'William Henry Harrison',
+    'Charles City County',
+    ' 3/4/1841'
+  ],
+  <String>[
+    'Martin Van Buren',
+    'Kinderhook New',
+    ' 3/4/1837'
+  ],
+  <String>[
+    'Zachary Taylor',
+    'Barboursville',
+    ' 3/4/1849'
+  ],
+  <String>[
+    'John Tyler',
+    'Charles City County',
+    ' 4/4/1841'
+  ],
+  <String>[
+    'James Buchanan',
+    'Cove Gap',
+    ' 3/4/1857'
+  ],
+  <String>[
+    'James K. Polk',
+    'Pineville North',
+    ' 3/4/1845'
+  ],
+  <String>[
+    'Millard Fillmore',
+    'Summerhill New',
+    '7/9/1850'
+  ],
+  <String>[
+    'Franklin Pierce',
+    'Hillsborough New',
+    ' 3/4/1853'
+  ],
+  <String>[
+    'Andrew Johnson',
+    'Raleigh North',
+    ' 4/15/1865'
+  ],
+  <String>[
+    'Abraham Lincoln',
+    'Sinking Spring',
+    ' 3/4/1861'
+  ],
+  <String>[
+    'Ulysses S. Grant',
+    'Point Pleasant',
+    ' 3/4/1869'
+  ],
+  <String>[
+    'Rutherford B. Hayes',
+    'Delaware',
+    ' 3/4/1877'
+  ],
+  <String>[
+    'Chester A. Arthur',
+    'Fairfield',
+    ' 9/19/1881'
+  ],
+  <String>[
+    'James A. Garfield',
+    'Moreland Hills',
+    ' 3/4/1881'
+  ],
+  <String>[
+    'Benjamin Harrison',
+    'North Bend',
+    ' 3/4/1889'
+  ],
+  <String>[
+    'Grover Cleveland',
+    'Caldwell New',
+    ' 3/4/1885'
+  ],
+  <String>[
+    'William McKinley',
+    'Niles',
+    ' 3/4/1897'
+  ],
+  <String>[
+    'Woodrow Wilson',
+    'Staunton',
+    ' 3/4/1913'
+  ],
+  <String>[
+    'William H. Taft',
+    'Cincinnati',
+    ' 3/4/1909'
+  ],
+  <String>[
+    'Theodore Roosevelt',
+    'New York City New',
+    ' 9/14/1901'
+  ],
+  <String>[
+    'Warren G. Harding',
+    'Blooming Grove',
+    ' 3/4/1921'
+  ],
+  <String>[
+    'Calvin Coolidge',
+    'Plymouth',
+    '8/2/1923'
+  ],
+  <String>[
+    'Herbert Hoover',
+    'West Branch',
+    ' 3/4/1929'
+  ],
+  <String>[
+    'Franklin D. Roosevelt',
+    'Hyde Park New',
+    ' 3/4/1933'
+  ],
+  <String>[
+    'Harry S. Truman',
+    'Lamar',
+    ' 4/12/1945'
+  ],
+  <String>[
+    'Dwight D. Eisenhower',
+    'Denison',
+    ' 1/20/1953'
+  ],
+  <String>[
+    'Lyndon B. Johnson',
+    'Stonewall',
+    '11/22/1963'
+  ],
+  <String>[
+    'Ronald Reagan',
+    'Tampico',
+    ' 1/20/1981'
+  ],
+  <String>[
+    'Richard Nixon',
+    'Yorba Linda',
+    ' 1/20/1969'
+  ],
+  <String>[
+    'Gerald Ford',
+    'Omaha',
+    'August 9/1974'
+  ],
+  <String>[
+    'John F. Kennedy',
+    'Brookline',
+    ' 1/20/1961'
+  ],
+  <String>[
+    'George H. W. Bush',
+    'Milton',
+    ' 1/20/1989'
+  ],
+  <String>[
+    'Jimmy Carter',
+    'Plains',
+    ' 1/20/1977'
+  ],
+  <String>[
+    'George W. Bush',
+    'New Haven',
+    ' 1/20, 2001'
+  ],
+  <String>[
+    'Bill Clinton',
+    'Hope',
+    ' 1/20/1993'
+  ],
+  <String>[
+    'Barack Obama',
+    'Honolulu',
+    ' 1/20/2009'
+  ],
+  <String>[
+    'Donald J. Trump',
+    'New York City',
+    ' 1/20/2017'
+  ],
```

Comment:
this is a really good example of much less readable code compared to the original.

Reply:
yeah that case doesn't look great. It would seem consistent to have the formatter only force list literals to each have their own line if there is a trailing comma.

Reply:
This was a bug on my part. I'm pushed a fixed version that doesn't mess up any of these list literals.